spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -125,5 +127,41 @@
         RUBY
       end
     end
+
+    context "when `AllowedPatterns: ['success']`" do
+      let(:cop_config) do
+        { 'Prefixes' => %w[when with], 'AllowedPatterns' => [/success$/] }
+      end
+
+      it 'finds context without `success` at the ending' do
+        expect_offense(<<-RUBY)
+          context 'when the display name not present' do
+                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Context description is not descriptions of allowed patterns.
+          end
+        RUBY
+      end
```

Comment:
I find this behavior confusing. The string starts with `when`, which is an allowed prefix.

I may not have thought this through, but it seems like an offense should be called only when **none** of the `Prefixes` match **and none** of the `AllowedPatterns` match.